### PR TITLE
Update to RxJS 6 imports and pipeable operators

### PR DESCRIPTION
**Pull request by tibetsprague**

### CODE REVIEW

File: src/combineLatestObject.js [hunk 1]
```diff
@@ -1,9 +1,7 @@
 // @flow
 
-import type { Observable } from 'rxjs/Observable'
-import { from } from 'rxjs/observable/from'
-import { combineLatest } from 'rxjs/observable/combineLatest'
-import { map as map$ } from 'rxjs/operators/map'
+import { combineLatest, from, Observable } from "rxjs";
+import { map as map$} from "rxjs/operators";
```
Comment (radex) [inline src/combineLatestObject.js]: why? this breaks optimizations, as the entirety of RxJS will be bundled with the app (when using Metro/RN)

**Comment by tibetsprague** [comment]
Because rxjs/observable/combineLatest and rxjs/operators/map don't exist as import paths in rxjs 6 and require installing the rxjs-compat package to work, which isn't a dependency of this package and adds to the bundle size: https://rxjs-dev.firebaseapp.com/guide/v6/migration

**Comment by radex** [comment]
Huh, that doesn’t seem right. I will investigate this in ~2 weeks

**Comment by joshfarrant** [comment]
@radex Any update on this? I've just run into this issue too and came here to open the same PR.

**Comment by radex** [comment]
Superseded by https://github.com/Nozbe/withObservables/pull/93

**Pull request closed without merging**